zeebe/test-util/src/main/java/io/camunda/zeebe/test/util/record/CompactRecordLogger.java

```diff
@@ -920,6 +922,12 @@ private String summarizeVariableDocument(final Record<?> record) {
             formatTenant(value));
   }
 
+  private String summarizeClusterVariable(final Record<?> record) {
+    final var value = (ClusterVariableRecordValue) record.getValue();
+    return "%s->%s for <tenant [%s]>"
```

Comment:
🔧 Since the `formatTenant` method already prints `(tenant: %s)` when the tenant is not DEFAULT, you could simplify the template:
```suggestion
    return "%s->%s%s"
```